vpdq/cpp/pdq/README.md

```diff
@@ -0,0 +1,5 @@
+Vendorized PDQ CPP implementation
```

Comment:
What does Vendorized mean?

Reply:
I meant to write [vendoring](https://stackoverflow.com/questions/26217488/what-is-vendoring)! Will update in all places.